src/Encamina.Enmarcha.AspNet.OpenApi.Swashbuckle/VersionSwaggerGenOptions.cs

```diff
@@ -8,5 +8,10 @@ public class VersionSwaggerGenOptions
     /// <summary>
     /// Gets or sets the title for the OpenAPI document when using version support.
     /// </summary>
-    public string Title { get; set; } = @"REST API";
-}
+    public required string Title { get; set; } = @"REST API";
+
+    /// <summary>
+    /// Gets or sets the description for the OpenAPI document.
```

Comment:
Maybe add the 'when using version support' sentence that has the property title too?

Reply:
Fixed!!